tests/core/fixtures.py

```diff
@@ -83,6 +84,57 @@ def shard_chain():
     return chain(shard_chaindb)
 
 
+@pytest.fixture
+def shard_chain_without_block_validation(funded_addr):
+    """
+    Return a Chain object containing just the genesis block.
+
+    This Chain does not perform any validation when importing new blocks.
+
+    The Chain's state includes one funded account specified by the `funded_addr` argument.
+
+    You can then deploy contract to the funded account.
+    """
+    shard_chaindb = BaseChainDB(get_db_backend(), state_backend_class=FlatTrieBackend)
+    overrides = {
+        'import_block': import_block_without_validation,
+        'validate_block': lambda self, block: None,
+    }
+    klass = Chain.configure(
+        name='TestShardChainWithoutBlockValidation',
+        vm_configuration=(
+            (constants.GENESIS_BLOCK_NUMBER, ShardingVM),
+        ),
+        **overrides,
+    )
+    initial_balance = 100000000
+    genesis_params = {
+        'block_number': constants.GENESIS_BLOCK_NUMBER,
+        'difficulty': constants.GENESIS_DIFFICULTY,
+        'gas_limit': constants.GENESIS_GAS_LIMIT,
+        'parent_hash': constants.GENESIS_PARENT_HASH,
+        'coinbase': constants.GENESIS_COINBASE,
+        'nonce': constants.GENESIS_NONCE,
+        'mix_hash': constants.GENESIS_MIX_HASH,
+        'extra_data': constants.GENESIS_EXTRA_DATA,
+        'timestamp': 1501851927,
+        'state_root': decode_hex(
+            '0x9d354f9b5ba851a35eced279ef377111387197581429cfcc7f744ef89a30b5d4')
+    }
+    genesis_state = {
+        funded_addr: {
+            'balance': initial_balance,
+            'nonce': 0,
+            'code': b'',
+            'storage': {},
+        }
+    }
+    chain = klass.from_genesis(shard_chaindb, genesis_params, genesis_state)
+    chain.funded_address = funded_addr
```

Comment:
Instead of storing this on the `chain` object, can we just use the` funded_address` fixture in any place where we need access to this address.  I'm worried that setting a precedent for tacking values onto objects will lead to confusing tests down the road when it's hard for someone to differentiate what is a formal `Chain` API and what is just part of the test setup.